src/components/date-picker/date-picker.tsx

```diff
@@ -284,7 +285,6 @@ export class DatePicker {
         );
         event.stopPropagation();
         if (this.type !== 'datetime' && this.type !== 'time') {
-            this.textField.blur();
```

Comment:
This had no effect as the textfield is not an input field but a limel-input-field.